src/main/java/fi/aalto/cs/apluscourses/intellij/actions/SubmitExerciseAction.java

```diff
@@ -301,6 +307,54 @@ private void trySubmit(@NotNull Project project)
     logger.debug("Finished submitting exercise");
   }
 
+  /**
+   * Submits a tutorial.
+   */
+  public void submitTutorial(@NotNull Project project,
+                             @NotNull TutorialViewModel tutorialViewModel) {
```

Comment:
I don't think this belongs here. If you want to have it in the same place as `trySubmit`, consider moving them both out of the action.